src/components/Tabs.js

```diff
@@ -58,6 +58,7 @@ class Tabs extends Component {
                     handleSelect,
                     isActive: child.props.to === selectedTab,
                     activeStyle: activeLinkStyle,
+                    disableInlineStyles: child.props.disableInlineStyles !== undefined ? child.props.disableInlineStyles : disableInlineStyles,
```

Comment:
What would be a use case for overriding this setting per TabLink/TabContent? I'd keep it configurable only on the main Tabs component.